java/templates/s2i/assemble

```diff
@@ -119,7 +119,7 @@ function build_maven() {
   local app_dir=$2
 
   # Default args: no tests, if a module is specified, only build this module
-  local maven_args=${MAVEN_ARGS:-package -DskipTests -e -Dfabric8.skip=true -B}
+  local maven_args=${MAVEN_ARGS:-package -DskipTests -Dmaven.javadoc.skip=true -Dmaven.site.skip=true -Dmaven.source.skip=true -Djacoco.skip=true -Dcheckstyle.skip=true -Dfindbugs.skip=true -Dpmd.skip=true -Dfabric8.skip=true -e -B}
```

Comment:
makes sense, but you need to return fish-pepper to pick it up in the templates directory.

Reply:
Duh! Yes, of course - sorry. Oh the joys of checked in generated code :smile: Now done, better? I've now also removed the fix for the broken build from the PR, so that doesn't get mixed up.